scripts/aboutus.js

```diff
@@ -94,144 +110,128 @@ const teamMembers = [
     ),
 ];
 
-
-const createTabs = (teamMembers) => {
+const createTabs = (teamMembers, introContent) => {
     const tabsContainer = document.querySelector(".team-tabs");
+    let lastActiveTab = null;
 
     teamMembers.forEach((person) => {
         const tab = document.createElement("div");
         tab.classList.add("tab");
         tab.textContent = person.tab;
 
-        
         tab.addEventListener("click", () => {
-            setActiveTab(tab);
-            updateTeamContent(person); 
+            if (lastActiveTab === tab) {
+                lastActiveTab = null;
+                setActiveTab(null);
+                updateTeamContent(introContent);
+            } else {
+                lastActiveTab = tab;
+                setActiveTab(tab);
+                updateTeamContent(person);
+            }
         });
-
         tabsContainer.appendChild(tab);
     });
 };
 
-
 const setActiveTab = (activeTab) => {
     document.querySelectorAll(".tab").forEach((tab) => tab.classList.remove("active"));
```

Comment:
in my humble opinion, you can query two classes at the same time,
.querySelect(".tab.active")

if one element has this active, .remove() the class.